### DISCUSSION TITLE
Errors thrown by powershell related to the add-member command

### DISCUSSION THREAD

**Discussion by Diet1976** [MartijnvanGeffen13/Volvo4evcc#35] [Q&A] [answered]
In the log there is the use of the parameter "force" with the add-member command recommended. Would it be useful, because the command always throws errors in my log? Thank you for your comment!
The log-entry: "Cannot add a member with the name "EvccStatus" because a member with
     | that name already exists. To overwrite the member anyway, add the Force
     | parameter to your command"

**Comment by MartijnvanGeffen13** [comment]
Hi @Diet1976 

What branch are you running (Main or multiplecarsimpler) and what version (See Volvo4evcc.psd1), I recognise the bug and think i fixed it in the 1.8.1 release of main or 1.10.1 release of multiplecarsimpler branch. 

If you still see it with these new versions let me know so i can investigate

**Reply by Diet1976** [reply]
Hey, thanks for the fast reply! I use the 1.5. I try the 1.8.1! Today in the morning I reveived a new Otp-Code per Mail without initiating. How often do I have to refresh the Code and do I have to put it manually in the .config-File or what does the Set-VolvoAuthenticate-Function do? Regards, Dietmar

**Reply by MartijnvanGeffen13** [reply, marked as answer]
hi ,

for the refresh i am not 100% sure as i have tryed looking at the refresh token but it seems to no contain expiration date. but following normal oauth flow it should be valid for 90 days. 

I have run for about 2,5 weeks without OTP (and with the old solution same setup 4 weeks) but now due to the code updates i often reinitiate due to updates as i test with my prod car. 

For you second question , 
If you run windows as host you can just start a 2nd Powershell import the module and run set-volvoconfig -otptoken "value"

If on linux starting a second shell remotely takes some more knowledge hence winscp connection and edit the config file is way easier. Or if you started via contab just use a file editor via remote session.  but if you want to on a remote session start PWSH , import the module and run the set-volvoconfig -otptoken "value" will also work.
